Country data card

weberjavi opened the pull request:
This PR adds the selected country data on the sidebar when selecting a country.
[PIVOTAL](https://www.pivotaltracker.com/story/show/171812784) | [DESIGN](https://projects.invisionapp.com/d/main#/console/17484384/409039745/preview)

![Kapture 2020-03-30 at 17 24 21](https://user-images.githubusercontent.com/6906348/77930623-67858880-72ab-11ea-9120-4332f2cce109.gif)

kfilip94 commented:
checking :eye: :eye: :eye:

kfilip94 commented:
- What about the mobile designs?

- It'd be nice to use `<sup>` tag to render 2 as a superscript element:
![image](https://user-images.githubusercontent.com/15097138/77942987-6573f780-72b4-11ea-9cce-aca91eca7c1f.png)

- Can we tweak the format of the values somehow, especially for small countries?
![image](https://user-images.githubusercontent.com/15097138/77942893-3eb5c100-72b4-11ea-9755-eff55b6a48b5.png)


- Country labels don't load properly on the local scene after a refresh. It's not strictly connected with this PR, I'll put it as a separate task to the Pivotal.
![image](https://user-images.githubusercontent.com/15097138/77943894-e1bb0a80-72b5-11ea-865d-f85f7cb45a16.png)

kfilip94 commented on src/components/country-data-card/country-data-card-styles.module.scss:
```diff
@@ -0,0 +1,45 @@
+  flex-direction: column;
+  width: $sidebar-width;
+  overflow: hidden;
+  color: white;
```
In case we would like to make our white less white in the future 
```suggestion
  color: $white;
```

kfilip94 commented on src/components/country-entry-card/country-entry-card.js:
```diff
@@ -1,12 +1,12 @@
 const CountryEntryCard = props => {
   const { changeUI, changeGlobe, sceneMode } = props;
-  const handleSceneModeChange = () => changeUI({ sceneMode: sceneMode === 'global' ? 'local' : 'global' });
+  const handleSceneModeChange = () => changeUI({ sceneMode: sceneMode === GLOBAL_SCENE ? LOCAL_SCENE : GLOBAL_SCENE });
```
Neat, thanks for that! :sunglasses:

kfilip94 commented on src/constants/view-props.js:
```diff
@@ -1,4 +1,7 @@
\ No newline at end of file
+};
+
+export  const LOCAL_SCENE = 'local';
```
```suggestion
export const LOCAL_SCENE = 'local';
```

kfilip94 commented:
Hey @weberjavi, the code looks nice, the functionality works smoothly. It requires a few tweaks, please check my comments above. Great job overall! :+1:

weberjavi commented:
@gretacv any thoughts about this comment? How should we treat this. Because we are not specifying the unit of the data, should we add `people` to the number? how should it behave when there are less than a million?
![image](https://user-images.githubusercontent.com/6906348/78009723-bbdb4780-7341-11ea-8410-c3e6f0d19ea3.png)

weberjavi commented:
@kfilip94 I've addressed some of the comments.
Regarding the mobile styles 've asked for some clarifications on [this pivotal task](https://www.pivotaltracker.com/story/show/172077983), so I would prefer to handle it on another PR. Does it sounds ok?
Please check the updates.
